policyengine_us/parameters/gov/states/ca/tax/income/credits/earned_income/phase_out/rate.yaml

```diff
@@ -1,30 +1,30 @@
-description: EITC phase-out rate
+description: California EITC phase-out rate
 metadata:
   rate_unit: currency-USD
   rate_period: year
   threshold_unit: child
   type: single_amount
-  label: CA EITC phase-out rate
+  label: California EITC phase-out rate
   reference:
     - title: California Legal code REVENUE AND TAXATION CODE - RTC (2019)
       href: https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=RTC&sectionNum=17052
-    - title: CA EITC Form 3514 instructions (2022)
-      href: https://www.ftb.ca.gov/forms/2022/2022-3514-instructions.html
+    - title: CA EITC Form 3514 instructions (various years)
+      href: https://www.ftb.ca.gov/forms/202X/202X-3514-instructions.html
 brackets:
-  - amount:
-      2022-01-01: 0.0765
-    threshold:
-      2022-01-01: 0
-  - amount:
-      2022-01-01: 0.34
-    threshold:
-      2022-01-01: 1
-  - amount:
-      2022-01-01: 0.40
-    threshold:
-      2022-01-01: 2
-  - amount:
-      2022-01-01: 0.45
-    threshold:
-      2022-01-01: 3
+  - threshold:
+      2021-01-01: 0
```

Comment:
While you're updating these, could you change it to 2015? The law specifies it's been consistent since then.

Reply:
Done